qt/aqt/reviewer.py

```diff
@@ -155,6 +155,10 @@ def _initWeb(self):
             ],
             context=self,
         )
+
+        if gui_hooks.will_init_bottom():
```

Comment:
Filters need to pass in the first argument so they can be chained, so the hook needs to take an argument like 'proceed', and be called with

```
if not gui_hooks.toolbar_bottom_will_init(True):
  return
```